integration-tests/conftest.py

```diff
@@ -58,3 +56,18 @@ def loop_until(predicate, poll_sec=5, timeout_sec=120):
         time.sleep(poll_sec)
         ok = predicate()
     return ok
+
+
+@pytest.fixture(scope="session", autouse=True)
+def collect_selinux_denials():
+    """This fixture helps in catching selinux denials
+    in the system after tests are run."""
+    yield
+    options = "ausearch -m avc -m user_avc -m selinux_err -i".split()
+    result = logged_run(options, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    if "<no matches>" not in str(result.stdout):
+        lines = str(result.stdout).split("\n")
+        for line in lines:
+            words = line.split()
+            if "denied" in words:
+                assert "permissive=1" in words, "SELinux AVC denials found"
```

Comment:
You could simplify this a bit by passing `text=True` to `logged_run()`. This would be an equivalent:

```py
result = logged_run(options, capture_output=True, text=True)
for line in result.stdout.split("\n"):
    if "denied" in line:
        assert "permissive=1" in line, "SELinux AVC denials found"
```

Reply:
I am ok to use capture_output=True instead of subprocess.PIPE. Only reason for using the later was to avoid python version issues. I think capture_output is available in version > python 3.7. But it should not affect the tests, I will modify the command. Thanks!

Reply:
@m-horky Like I suspsected 'capture_output' is failing on RHEL8 tests(python 3.6), so I am reverting the change back to original commit.